contents/javascript/Javascript-framework-VueJs.md

```diff
@@ -11,18 +11,18 @@
 
 {{ booktitle | safe }}
 
-# VueJs
+# Introduction to Vue
 
 **Authors: [Chelsey Ong](https://github.com/chelseyong), [Lu Lechuan](https://github.com/LuLechuan)** <br>
 Reviewers: [Gilbert Emerson](https://github.com/emer7), [Ong Shu Peng](https://github.com/ongspxm)
 
 <box type="info">
-This article assumes the reader has a basic knowledge of HTML and JavaScript.
+This chapter assumes that the reader has a basic knowledge of HTML and JavaScript.
 </box>
 
-## What is VueJs?
+## What is Vue?
 
-VueJs (also known as Vue) is an open-source [JavaScript framework](https://en.wikipedia.org/wiki/JavaScript_framework) for building user interfaces. It is designed to improve code quality and maintainability.
+>*Vue* (also known as Vue) is an open-source [JavaScript framework](https://en.wikipedia.org/wiki/JavaScript_framework) for building user interfaces. It is designed to improve code quality and maintainability.
```

Comment:
You've written `Vue (also known as Vue) is an open-source ...`
The words in the parentheses can be removed?